### PR TITLE
Some broker tests

### PR DESCRIPTION
**Pull request by booqoffsky**
I have to listen in order not to lose the message in PubSub

### CODE REVIEW

File: tests/test_broker.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+async def get_message(broker: AsyncBroker) -> BrokerMessage:
+    """
+    Get a message from the broker.
+
+    :param broker: async message broker.
+    :return: first message from listen method.
+    """
+    return await anext(broker.listen())
```
Comment (s3rius) [inline tests/test_broker.py:17]: This won't pass in python 3.7.

Comment (s3rius) [reply tests/test_broker.py:17]: To make it possible you can convert it to

```python
async for message in broker.listen():
    return message
```

Comment (booqoffsky) [reply tests/test_broker.py]: Fixed it

Comment (s3rius) [reply tests/test_broker.py]: Nice!

**Comment by s3rius** [comment]
Please rebase onto develop branch. I updated workflow policy and it should add CI in PRs now.

**Review by s3rius — APPROVED** [review]
Nice looking. Thanks!

**Pull request merged**